resources/overlap/hitbox.gd

```diff
@@ -3,3 +3,7 @@ extends Area2D
 class_name Hitbox
 
 @export var damage: int = 0
+
+func _on_body_entered(body):
+	if body is TileMap:
```

Comment:
Souldnt be done in this script because it would apply to all hitboxes i.e. enemies as well.

Reply:
Ok